AnnoyancesFilter/sections/subscriptions.txt

```diff
@@ -38,6 +38,7 @@ turkuazgazetesi.net,yenialanya.com,mansetaydin.com,seskocaeli.com,kocaelibarisga
 michalpasterski.pl##.newsletter-popup
 thestreet.com##.bcpContentTemplate
 fotospor.com##.alert-message
+orefolder.net##.followme
```

Comment:
They are links to their social media. @zloyden

Reply:
@Yuki2718 can you please translate this?
```
の最新情報をお届けします
```
As I understand, it's just their social links without any recommendations like `Find us here...`.
@jellizaveta we can move this rule to the `annoyances.txt`.

Reply:
Maybe "These bring you up-to-date information about OREFOLDER."

Reply:
Hm, it is for `annoyances.txt`.